### PR TITLE
FEAT: add image styling in markdown

### PR DESCRIPTION
**Pull request by Yash-1511**
In this change, i have added some image styling. 

Before:
Screenshot from 2024-02-13 12-23-33

After:
Screenshot from 2024-02-13 12-22-04

### CODE REVIEW

**Review by takatost — CHANGES_REQUESTED** [review]
Thank you so much for your contribution! 
There's an eslint error that needs to be fixed, could you please take a look at it?
 https://github.com/langgenius/dify/actions/runs/7888879799/job/21527443340?pr=2441

**Comment by Yash-1511** [comment]
Yeah... Sure i will check and update it.

**Comment by bowenliang123** [comment]
> Thank you so much for your contribution! There's an eslint error that needs to be fixed, could you please take a look at it? https://github.com/langgenius/dify/actions/runs/7888879799/job/21527443340?pr=2441

BTW, we could have an extra hint step for failed jobs of `Web Sytle Check` to tell the contributor what to do or how to fix them by using `yarn fix`.

File: web/yarn.lock [hunk 1]
```diff
@@ -5859,16 +5859,7 @@ resolve-pkg-maps@^1.0.0:
...
-  version "1.22.2"
-  resolved "https://registry.npmjs.org/resolve/-/resolve-1.22.2.tgz"
-  integrity sha512-Sb+mjNHOULsBv818T40qSPeRiuWLyaGMa5ewydRLFimneixmVy2zdivRl+AF6jaYPC8ERxGDmFSiqui6SfPd+g==
-  dependencies:
-    is-core-module "^2.11.0"
-    path-parse "^1.0.7"
-    supports-preserve-symlinks-flag "^1.0.0"
-
-resolve@^1.22.4:
+resolve@^1.1.7, resolve@^1.10.0, resolve@^1.22.1, resolve@^1.22.2, resolve@^1.22.4:
```
Comment (bowenliang123) [inline web/yarn.lock]: It seems unnecessary and brings no gains, as there is no change in `package.json`.

**Review by crazywoola — DISMISSED** [review]

**Review by crazywoola — APPROVED** [review]

**Review by takatost — APPROVED** [review]
LGTM

**Comment by bowenliang123** [comment]
How about just use `max-width` attribute instead?

**Comment by bowenliang123** [comment]
A hardcoded 250 in `width` and `height` may not be satisfied for different responsive or size of screens.

**Pull request merged**